src/semgrep_agent/semgrep_app.py

```diff
@@ -45,22 +45,43 @@ class Scan:
 class Sapp:
     url: str
     token: str
-    deployment_id: int
+    deployment_id: int = -1
+    deployment_name: str = ""
```

Comment:
Maybe it'd be better to initialize this and the ID to None? That way the type checker will remind us everywhere that these might not be available.